src/PowerShellEditorServices/Language/LanguageService.cs

```diff
@@ -569,6 +591,55 @@ public FindOccurrencesResult FindSymbolsInFile(ScriptFile scriptFile)
             return foundDefinition;
         }
 
+        private Ast FindSmallestStatementAst(ScriptFile scriptFile, int lineNumber, int columnNumber)
+        {
+            var asts = scriptFile.ScriptAst.FindAll(ast =>
+            {
+                if (!(ast is StatementAst))
+                {
+                    return false;
+                }
+
+                var scriptExtent = ast.Extent;
+                if (scriptExtent.StartLineNumber > lineNumber || scriptExtent.EndLineNumber < lineNumber)
+                {
+                    return false;
+                }
+
+                if (scriptExtent.StartLineNumber == lineNumber)
+                {
+                    return scriptExtent.StartColumnNumber <= columnNumber;
+                }
+
+                if (scriptExtent.EndLineNumber == lineNumber)
+                {
+                    return scriptExtent.EndColumnNumber >= columnNumber;
+                }
+
+                return true;
+            }, true);
+
+            if (asts == null || asts.Count() == 0)
```

Comment:
IEnumerable<T>.Any() is way faster than IEnumerable<T>.Count() because it checks if there's at least one item rather than iterating over the whole collection to get the full count.  Doesn't matter at all in this case, just a helpful tip to pass along :)